### ISSUE TITLE
There is something going wrong in parsing meaning with extra sub meaning lists

### ISSUE THREAD

**Issue by Praveena0989** [Suyash458/WiktionaryParser#11]
I was trying to parse word `and` and then it gave me weird results which include extra list item for a single meaning merged into one and its repeating.

**Comment by mechhariom** [comment]
Same problem i have

**Comment by Suyash458** [comment]
Yeah I've noticed those problems too. This project was a hobby during college so I'm not actively maintaining it right now. I'll be making some changes to parsing in the future but if you have any inputs, feel free to send a pull request.

**Comment by Suyash458** [comment]
Seems fixed in 5952be49ecc6eb13aed74b5c06f0e56728c631b0
